tortoise/expressions.py

```diff
@@ -12,9 +12,7 @@
 class F(Field):  # type: ignore
     @classmethod
     def resolver_arithmetic_expression(
-        cls,
-        model: "Type[Model]",
-        arithmetic_expression_or_field: Union[ArithmeticExpression, Field],
+        cls, model: "Type[Model]", arithmetic_expression_or_field: Term,
     ) -> Tuple[Union[ArithmeticExpression, Field], Optional[Field]]:
```

Comment:
Should the response type not also change to `Tuple[Term, Optional[Field]]`?

Reply:
Yeah,my negligence